examples/gno.land/r/gnoland/faucet/faucet.gno

```diff
@@ -34,13 +35,11 @@ func Transfer(to std.Address, send int64) string {
 
 	var sendCoins std.Coins
 
-	if send <= limit.AmountOf("ugnot") {
-
+	// limit the pre request
```

Comment:
What do you think about simply sending the maximum limit when the requested amount is above the threshold, instead of denying the request?

You could simplify this code segment this way:

```go
// You can make this a separate helper method
getUgnotCoins := func(amount int64) std.Coins {
    return std.Coins { std.Coin { Denom: "ugnot", Amount: amount } }
}

var (
    maxAmount = gLimit.AmountOf("ugnot")
    sendCoins = getUgnotCoins(maxAmount)
)

// Check if the requested amount 
 // is lower than the threshold
if send < maxAmount {
    sendCoins = getUgnotCoins(send)
}

// ...

```